app/webpacker/styles/_units.scss

```diff
@@ -1,21 +1,30 @@
 // units table
```

Comment:
Not sure where to add this but it's todo with the unit page.
When viewing the unit#show page on a small screen the side bar breaks above the lesson table which seems a bit odd
![Screenshot 2020-03-13 at 16 42 38](https://user-images.githubusercontent.com/9936028/76641738-fb382480-6549-11ea-802a-e1585dd13327.png)

Reply:
Ah yeah, I tried to make the table wider by placing it in a separate row underneath. I'll move it up to the same column as the text content and it should flow in the right order :+1: